docs/DeveloperGuide.md

```diff
@@ -427,7 +426,7 @@ e.g. "This is a sentence!" contains the word "This", "is", "a" and "sentence!
 
 ### 4.3. Memo and ContactedDate person attributes
 
-The address book `Memo` and `ContactedDate` are data fields, part of `Person`. `Memo` allow users to store miscellaneous information about a `Person`, while `ContactedDate` allow users to keep track of the last contacted date of a `Person`. `Memo` and `ContactedDate` are optional fields, that is, both can be empty.
+The address book `Memo` and `ContactedDate` are person attributes, part of `Person`. `Memo` allow users to store miscellaneous information about a `Person`, while `ContactedDate` allow users to keep track of the last contacted date of a `Person`. `Memo` and `ContactedDate` are optional attributes, i.e. both can be empty.
```

Comment:
Perhaps try "either can be empty" over "both can be empty"? I'm just nitpicking here so it's fine if you don't change